lib/froala-editor-sdk/file.rb

```diff
@@ -12,19 +12,18 @@ class File
             allowedExts: [".txt", ".pdf", ".doc", ".json", ".html"],
             allowedMimeTypes: [ "text/plain", "application/msword", "application/x-pdf", "application/pdf", "application/json","text/html" ]
         },
-        resize: nil
+        resize: nil,
+        file_upload_path: 'public/uploads/files',
+        file_access_path: '/uploads/'
     }
 
-    # Default upload path.
-    @default_upload_path = "public/uploads/files"
-
     # Uploads a file to the server.
     # Params:
     # +params+:: File upload parameter mostly is "file".
     # +upload_path+:: Server upload path, a storage path where the file will be stored.
     # +options+:: Hash object that contains configuration parameters for uploading a file.
     # Returns json object
-    def self.upload(params, upload_path = @default_upload_path, options = {})
+    def self.upload(params, options = {})
```

Comment:
I would like to keep the existing interface for methods.

Reply:
@stefanneculai It would be an incorrect fix as we would still have this field outside as we should be having this in the options which needs to be supplied. Thanks